src/contexts/navigator.tsx

```diff
@@ -25,7 +25,7 @@ export type NavigatorCache = {
  */
 const NavigatorMapContext = createContext<NavigatorCache | null>(null);
 
-function useMapContext() {
+function UseMapContext() {
```

Comment:
Note that eslint is requiring the use of uppercase for the `UseMapContext` due to its inclusion in the other functions.

eslint[react-hooks/rules-of-hooks](https://reactjs.org/docs/hooks-rules.html)